frontend/src/components/charger_frame.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button } from './Button';
+
+function ChargerFrame() {
+  const textContent = [
+    // { label: 'Charger Name', value: 'Sample A' },
+    { label: 'State', value: 'Available' },
+    { label: 'Meter Value', value: '123456' },
+    { label: 'Max Power', value: '40A' },
+    { label: 'Current Power', value: '0' },
+  ];
+
+  return (
+    <div
+      style={{
+        width: '300px',
+        border: '2px solid #000',
+        borderRadius: '8px',
+        padding: '20px',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        backgroundColor: '#f9f9f9',
+      }}
+    >
+      <div
+        style={{
+          marginBottom: '20px', // Larger gap for Charger Name
+          fontSize: '18px', // Larger font for Charger Name
+          fontWeight: 'bold', // Bold Charger Name
+        }}
+      >
+        <p style={{ margin: '0', textAlign: 'left' }}>
+          <strong>Charger Name:</strong> Sample A
+        </p>
+      </div>
+
+      {/*the data displayed below for State, Meter value etc are temporary, will replace
+    with real data later*/}
+      {textContent.map((item, index) => (
```

Comment:
The text content should have its own component, like a "TextRow" component. They will likely be reused later, so it would be good to have.